core/types/calls.cc

```diff
@@ -2089,7 +2089,7 @@ class Kernel_proc : public IntrinsicMethod {
     }
 } Kernel_proc;
 
-class enumerable_to_h : public IntrinsicMethod {
+class Enumerable_toH : public IntrinsicMethod {
```

Comment:
Is there a reason we want to keep the `_` in here or can we change this to `EnumerableToH` as well?

Reply:
We register intrinsics as `ClassName_methodName`. This is the `to_h` method on the `Enumerable` class.

This intrinsic is implemented by dispatching to `Sorbet::Private::Static.enumerable_to_h`, which is why we need the entry in generate_names.